'cnd down' crash

**Describe the bug**
`cnd down` crashed leaving the following trace:
```
cnd down
Deactivating your cloud native development environment...
warning: there are 2 cloud native development environments active in your current folder, using 'default/frontend'
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/okteto/cnd/cmd.findDevEnvironment(0xc000adfc00, 0x1, 0x1, 0x3a, 0x0, 0x0, 0x2, 0x2, 0x2)
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/cmd/exec.go:103 +0x502
github.com/okteto/cnd/cmd.executeDown(0x0, 0x0)
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/cmd/down.go:30 +0x84
github.com/okteto/cnd/cmd.Down.func1(0xc000a4a000, 0x2f7f7f8, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/cmd/down.go:20 +0x22
github.com/okteto/cnd/vendor/github.com/spf13/cobra.(*Command).execute(0xc000a4a000, 0x2f7f7f8, 0x0, 0x0, 0xc000a4a000, 0x2f7f7f8)
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/vendor/github.com/spf13/cobra/command.go:762 +0x473
github.com/okteto/cnd/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2f56e00, 0xc000adff88, 0x1007500, 0xc00009e058)
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/okteto/cnd/vendor/github.com/spf13/cobra.(*Command).Execute(0x2f56e00, 0x0, 0x0)
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/okteto/cnd/cmd.Execute()
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/cmd/root.go:57 +0x2d
main.main()
	/private/tmp/cnd-20190107-22280-8elod0/src/github.com/okteto/cnd/main.go:22 +0x20
```

Previous to this `cnd up` was killed because of the error the following error:
```
rpc error: code = Unknown desc = Error: No such container: 24c661d7152e1914841cbf5ed8c79f0b22d80dede38736367534283b8244fd19
```
There are no CND environments enabled when the down is launched, maybe this is what causes the 'index out of range' error. 

**Desktop (please complete the following information):**
 - Version 0.4.0

--- a/cmd/exec.go
+++ b/cmd/exec.go
@@ -94,6 +94,9 @@ func findDevEnvironment(mustBeRunning bool) (string, string, string, error) {
 	}
 
 	parts := strings.SplitN(deploymentFullName, "/", 3)
+	if len(parts) < 3 {
+		return "", "", "", fmt.Errorf("unable to access the cnd local state. Remove the folder '~/.cnd' and try again")
+	}
 	namespace := parts[0]
 	deploymentName := parts[1]
 	devContainer := parts[2]

